### ISSUE TITLE
JTidy converts HTML 5 Doctype declaration into HTML 2.0 Doctype declaration

### ISSUE THREAD

**Issue by smarkwal** [jtidy/jtidy#65]
When migrating from JTidy r938 to the latest JTidy 1.0.4, I noticed that HTML 5 Doctype declarations present in the input HTML code are now converted to HTML 2.0 Doctype declarations. This is a new behavior. While JTidy r938 did not have official support for HTML 5, it did not change an HTML 5 Doctype declaration that was present in the input.

HTML input:
```html
<!DOCTYPE html>
<html>
   [...]
</html>
```

HTML output with JTidy r938:
```html
<!DOCTYPE html>
<html>
   [...]
</html>
```

HTML output with JTidy 1.0.4:
```html
<!DOCTYPE html PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html>
   [...]
</html>
```

Here is the source code that I use to parse and reformat the HTML code:

```java
public static String formatHtmlCode(InputStream htmlStream) {
	Tidy tidy = new Tidy();
	tidy.setIndentContent(true);
	tidy.setErrout(new PrintWriter(NullOutputStream.NULL_OUTPUT_STREAM)); // do not write messages to STDERR
	ByteArrayOutputStream buffer = new ByteArrayOutputStream();
	tidy.parse(htmlStream, buffer);
	return buffer.toString();
}
```

Is this the expected behavior and I'm missing something, maybe some change between r938 and 1.0.4 which requires changes in my code? Or is it a bug? Is something broken with the automatic detection of the Doctype?

Thanks in advance for any help on this topic.

Best regards,
Ste

**Comment by haumacher** [comment]
I added rudimentary HTML5 support some days ago - of cause, it was never intended to replace the doctype. So I would classify this as a bug.

**Comment by haumacher** [comment]
@smarkwal Can you verify that the changes in #66 fix your problem? If that the case, I could release a new version with a fix.

**Comment by smarkwal** [comment]
Hi @haumacher 

I can confirm that my previously failing unit tests now pass when I apply your fix from #66.
Thank you very much for your help!

**Comment by haumacher** [comment]
@smarkwal I released an update containing the fix. If you like, you can update your dependencies to

```
<dependency>
    <groupId>com.github.jtidy</groupId>
    <artifactId>jtidy</artifactId>
    <version>1.0.5</version>
</dependency>
```
